wasi-sdk.cmake

```diff
@@ -16,23 +16,14 @@ endif()
 
 set(CMAKE_C_COMPILER ${WASI_SDK_PREFIX}/bin/clang${WASI_HOST_EXE_SUFFIX})
 set(CMAKE_CXX_COMPILER ${WASI_SDK_PREFIX}/bin/clang++${WASI_HOST_EXE_SUFFIX})
-set(CMAKE_AR ${WASI_SDK_PREFIX}/bin/llvm-ar${WASI_HOST_EXE_SUFFIX} CACHE STRING "wasi-sdk build")
-set(CMAKE_RANLIB ${WASI_SDK_PREFIX}/bin/llvm-ranlib${WASI_HOST_EXE_SUFFIX} CACHE STRING "wasi-sdk build")
-set(CMAKE_C_COMPILER_TARGET ${triple} CACHE STRING "wasi-sdk build")
-set(CMAKE_CXX_COMPILER_TARGET ${triple} CACHE STRING "wasi-sdk build")
-set(CMAKE_C_FLAGS "-v" CACHE STRING "wasi-sdk build")
-set(CMAKE_CXX_FLAGS "-v -std=c++11" CACHE STRING "wasi-sdk build")
```

Comment:
Weird.. why would we ever want `-v` to be the default cflags..

Reply:
Debugging code that got left behind...